pallets/ah-ops/src/lib.rs

```diff
@@ -260,6 +314,44 @@ pub mod pallet {
 
 			Self::do_unreserve_crowdloan_reserve(block, depositor, para_id).map_err(Into::into)
 		}
+
+		/// Try to migrate a parachain sovereign child account to its respective sibling.
+		///
+		/// Takes the old and new account and migrates it only if they are as expected. An event of
+		/// `SovereignMigrated` will be emitted if the account was migrated successfully.
+		///
+		/// Callable by any signed origin.
+		#[pallet::call_index(3)]
+		#[pallet::weight(<T as Config>::WeightInfo::migrate_parachain_sovereign_acc())]
+		pub fn migrate_parachain_sovereign_acc(
+			origin: OriginFor<T>,
+			from: T::AccountId,
+			to: T::AccountId,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			Self::do_migrate_parachain_sovereign_derived_acc(&from, &to, None).map_err(Into::into)
+		}
+
+		/// Try to migrate a parachain sovereign child account to its respective sibling.
+		///
+		/// Takes the old and new account and migrates it only if they are as expected. An event of
+		/// `SovereignMigrated` will be emitted if the account was migrated successfully.
+		///
+		/// Callable by any signed origin.
+		#[pallet::call_index(5)]
+		#[pallet::weight(<T as Config>::WeightInfo::migrate_parachain_sovereign_derived_acc())]
+		pub fn migrate_parachain_sovereign_derived_acc(
+			origin: OriginFor<T>,
+			from: T::AccountId,
+			to: T::AccountId,
+			derivation: (T::AccountId, DerivationIndex),
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			Self::do_migrate_parachain_sovereign_derived_acc(&from, &to, Some(derivation))
+				.map_err(Into::into)
+		}
```

Comment:
are we gonna really need them? aren't we having with this two impls of sovereign accounts mapping?

Reply:
Yeah, just noticed that we won't need these extrinsics for sure. Since this time we are translating the accounts automatically.